framework/widgets/Pjax.php

```diff
@@ -117,7 +121,8 @@ public function init()
     public function run()
     {
         if (!$this->requiresPjax()) {
-            echo Html::endTag('div');
+            $tag = ArrayHelper::remove($this->options, 'tag', 'div');
```

Comment:
on the previous call the `$this->options` variable is saved on a temporary variable `$options` so `$this->options` is actually unaffected.